extensions/amp-consent/0.1/consent-info.js

```diff
@@ -58,16 +59,34 @@ export const CONSENT_ITEM_STATE = {
   // {unknown, accepted, rejected}
 };
 
+/**
+ * Used in ConsentInfoDef
+ * @enum {number}
+ */
+export const CONSENT_STRING_TYPE = {
```

Comment:
Third party code will need to import this enum. Better not to keep it under the consent extension. Please refer to `CONSENT_POLICY_STATE`